contrib/caldav/README.md

```diff
@@ -63,3 +63,104 @@ added or deleted. It then
 Note: Since calcurse does not use unique identifiers for items, it cannot keep
 track of moved/edited items. Thus, modifying an item is equivalent to deleting
 the old item and creating a new one.
+
+OAuth2 Authentication
+---------------------
+calcurse-caldav also has support for services requiring OAuth2 authentication
+such as Google Calendar. Note that you can only have a single calendar synced
+at any given time, regardless of authentication method. To enable OAuth2 you
+will need to:
+
+* Change *AuthMethod* from "*basic*" to "*oauth2*" in your config file
+* Fill in the appropriate settings in your config file: "*ClientID*",
+ "*ClientSecret*", "*Scope*", and "*RedirectURI*". These can be obtained from
+ the API provider. (See below for Google Calendar)
+* Install *oauth2client* for Python 3 using *pip* or your distro's package
+manager
+
+Example for distro where Python 3 is the default (e.g., Arch Linux):
+```
+pip install --user oauth2client
+```
+Example for distro where Python3 is not the default (e.g., Fedora 26):
+
+```
+pip3 install --user oauth2client
+```
```

Comment:
Can't we always advise to use `pip3` here? Then maybe just put this into the bullet point above, like "Install *oauth2client* for Python 3 using *pip* (e.g. `pip3 install --user oauth2client`) or your distribution's package manager."